docs/modules/ROOT/pages/index.adoc

```diff
@@ -503,21 +513,188 @@ In each of the above sections, the following changes MUST be applied:
+.. If the transitive trust relationship exists and the credential has not been revoked, proceed to step 2.
+.. If neither relationship can be demonstrated, do not proceed. The *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+. Does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the _<<_named_actor,named actor’s>> credential issuer_ to issue valid credentials?
+.. Is there a direct trust relationship with the _<<_named_actor,named actor’s>> credential issuer?_ If so, proceed to step 3.
+.. Is there a transitive trust relationship with the _<<_named_actor,named actor’s>> credential issuer_ via its _credential issuer?_ (In other words, does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the _<<named_actor,named actor’s>> credential issuer_ to issue valid credentials?) If so, proceed to step 3.
+.. If neither relationship can be demonstrated, do not proceed. The *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+. Has the _credential issuer_ issued a revocation for the _<<_named_actor,named actor’s>>_ credential?
+.. If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+.. If no such revocation has been issued, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *trusted.*
+
+=== Threats to trust model
+
+_This section is non-normative._
+
+This section enumerates a number of potential attacks on the *<<_identity_assertion,identity assertion>>* trust model. If concrete guidance to mitigate or prevent a specific attack is available, that guidance should be incorporated as specific normative requirements elsewhere in this specification and referenced here.
+
+==== Replay attacks
+
+* Could an attacker extract the *<<_identity_assertion,identity assertion>>* out of one _<<_c2pa_asset,C2PA asset>>_ and embed it into another _<<_c2pa_asset,C2PA asset>>?_
+* Is there anything that makes the *<<_identity_assertion,identity assertion>>* unique to that _<<_c2pa_asset,C2PA asset>>?_
+* Does the _<<_identity_assertion_consumer,identity assertion consumer>>_ need to do any checks to make sure that a replay attack is not occurring?
+
+An attacker could, in theory, extract an *<<_identity_assertion,identity assertion>>* and place it into a new _<<C2PA Manifest>>_ as part of an unrelated _<<_c2pa_asset,C2PA asset>>._
+
+To prevent this attack, a valid *<<_identity_assertion,identity assertion>>* must contain a `signer_payload.referenced_assertions` include a _<<_hard_binding,hard binding>>_ assertion that properly describes the _<<_c2pa_asset,C2PA asset>>._ A compliant _<<_identity_assertion_consumer,identity assertion consumer>>_ should detect that the _<<_hard_binding,hard binding>>_ assertion referenced by the original *<<_identity_assertion,identity assertion>>* does not match the attacker’s _<<_c2pa_asset,C2PA asset>>_ and fail validation.
+
+==== Name collisions
+
```

Comment:
Overview:

For an identity to have value, it must be unique. An attacker may try to cause confusion for the content consumer by trying to impersonate a known, trusted identity. One method for attempting to impersonate a trusted identity is to attempt to create an identity with the same name through a second identity provider. If the content consumer doesn't know which identity provider the trusted identity uses, then the attacker may be able to convince the content consumer that their content is from the trusted identity. In addition to intentional collisions, there could also be unintentional collisions due to the fact that thousands of people can have the same name. Therefore, there needs to be methods for distinguishing people of the same name both within a single identity provider and across identity providers.

Reply:
Incorporated this language. Thank you, @puhley!